### DISCUSSION TITLE
Send digital product automatically on successful order

### DISCUSSION THREAD

**Discussion by scarereeperr** [Steffo99/greed#97] [Q&A] [answered]
I sell digital products, and I would like a feature so that when the order is complete it automatically sends them a license key to the product they bought. Is there any way to achieve this?

**Comment by Steffo99** [comment, marked as answer]
You should be able to add code for your use case in this function:
https://github.com/Steffo99/greed/blob/3cfe0251d126c5342715d03edc3ba839a2b79acf/worker.py#L697-L710

**Reply by scarereeperr** [reply]
Also something I noticed is when I tried to order something, instead of it just refusing saying I don't have enough money, it went into the negatives, is there any way to fix that?

**Reply by Steffo99** [reply]
That's a different issue, I'm answering in #99.

**Reply by Steffo99** [reply]
I'm setting this as answer, since there hasn't been any activity here for more than an year.
